controllers/istiorevision/istiorevision_controller.go

```diff
@@ -235,6 +235,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 		// namespaced resources
 		Watches(&corev1.ConfigMap{}, ownedResourceHandler).
 		Watches(&appsv1.Deployment{}, ownedResourceHandler). // we don't ignore the status here because we use it to calculate the IstioRevision status
+		// nolint:staticcheck
```

Comment:
Is this the only change?

Reply:
yes, it's needed for the linter to pass with new upstream settings